src/wasm-features.h

```diff
@@ -47,11 +47,16 @@ struct FeatureSet {
     TypedContinuations = 1 << 16,
     SharedEverything = 1 << 17,
     FP16 = 1 << 18,
+    BulkMemoryOpt = 1 << 19, // Just the memory.copy and fill operations
+    // This features is a no-op for compatibility. Having it in this list means
+    // that we can automatically generate tool flags that set it, but otherwise
+    // it does nothing. Binaryen always accepts LEB call-indirect encodings.
+    CallIndirectOverlong = 1 << 20,
```

Comment:
Please link to the tool-conventions doc that explains what these are, as all the other features are easily found on the main wasm website, but not these.

Reply:
Actually these are missing from the tool-conventions [doc](https://github.com/WebAssembly/tool-conventions/blob/58cdfb4dbbd801516ee384821c0adb800d03732e/Linking.md?plain=1#L416), we should fix that.

Reply:
https://github.com/WebAssembly/tool-conventions/pull/242